test/e2e/apimachinery/flowcontrol.go

```diff
@@ -154,6 +158,10 @@ var _ = SIGDescribe("API priority and fairness", func() {
 	// the two clients and not allow one client to drown out the other despite
 	// having a higher QPS.
 	ginkgo.It("should ensure that requests can't be drowned out (fairness)", func() {
+		// See https://github.com/kubernetes/kubernetes/issues/96710
+		ginkgo.Skip("skipping test until flakiness is resolved")
+		return
```

Comment:
I think `ginkgo.Skip` implicitly exits, but doesn't hurt to return early also for now

Reply:
yeah, I wasn't entirely sure; the ginkgo docs seem to suggest calling `Skip` is sufficient, but the other e2e tests in k/k seem to return :sweat_smile:

Reply:
you were right, we don't need a return here (in fact, govet complains about unreachable code)